### PR TITLE
add docker-compose postgres. add team on seed. update README.md.

### PR DESCRIPTION
**Pull request by jonkurtis**
Created a local docker-compose.yml with postgres service for local development. Updated `@/lib/db/seed.ts` to include a team for the test user. Updated README.md with docker-compose usage and clearer setup instructions for database along with test user login credentials.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+services:
+  postgres:
+    image: postgres:15
```
Comment (webda2l) [inline docker-compose.yml:3]:
```suggestion
    image: postgres:16
```

At least. 17 soon, https://www.postgresql.org/about/news/postgresql-17-rc1-released-2926/

File: lib/db/seed.ts [hunk 2]
```diff
@@ -1,64 +1,82 @@
...
+    role: "owner",
+  });
 
   await createStripeProducts();
 }
 
 seed()
   .catch((error) => {
-    console.error('Seed process failed:', error);
+    console.error("Seed process failed:", error);
```
Comment (leerob) [inline lib/db/seed.ts:76]: Can we reset the lockfile changes and the Prettier changes?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+services:
+  postgres:
+    image: postgres:16
```
Comment (jlucaso1) [inline docker-compose.yml:3]: Can we use postres:16-alpine?

File: lib/db/seed.ts [hunk 4]
```diff
@@ -39,17 +39,35 @@ async function createStripeProducts() {
...
+  const email = 'test@test.com';
+
+  const [user] = await db
+    .insert(users)
+    .values([
+      {
+        email: email,
+        // admin123
+        passwordHash:
+          '$2a$10$7JNtY2uXCS.QoyoH8i.mRu1g4sUtK5ausmU.5MjBVRCm.hVflCw0y',
```
Comment (jlucaso1) [inline lib/db/seed.ts:51]: it's better call `await hashPassword("admin123")`

**Review by jlucaso1 — APPROVED** [review]

**Comment by leerob** [comment]
https://github.com/leerob/next-saas-starter/commit/2ae775d349640064e62afbc1dbee9cc98386f45a

**Review by preechapon-cmd — APPROVED** [review]

**Pull request merged**